docs/user-guide/migrating-to-6.0.0.md

```diff
@@ -0,0 +1,52 @@
+# Migrating to v6.0.0
+
+ESLint v6.0.0 is the fifth major version release. We have made a few breaking changes in this release, but we expect that most users will be able to upgrade without any modifications to their build. This guide is intended to walk you through the breaking changes.
+
+The lists below are ordered roughly by the number of users each change is expected to affect, where the first items are expected to affect the most users.
+
+### Breaking changes for users
+
+- [`no-redeclare` rule now checks the redeclarations by `/* globals */` directive comments](#no-redeclare-and-comments)
+- [`no-redeclare` rule now checks the redeclarations with built-in globals by default](#no-redeclare-and-builtins)
```

Comment:
To avoid merge conflicts, I was thinking it might be better to create the migration guide separately rather than adding an entry in each breaking commit.

Reply:
I got it. I will remove the migration guide.